R/CompDb.R

```diff
@@ -59,7 +59,11 @@
 #'     to the database with parameter `x`.
 #'
 #'     For all other methods: a `CompDb` object.
-#'
+#' 
+#' @param flags flags passed to the SQLite database connection.
+#'     See \link[RSQLite]{SQLite}. Defaults to read-only, i.e. RSQLite::SQLITE_RO.
```

Comment:
Please use only markdown syntax in roxygen documentation: use `[SQLite()]` instead of `\link[RSQLite]{SQLite}`.

Reply:
Ah. I am behind the times I see. I have done the update.

Reply:
👍